go/tools/builders/env.go

```diff
@@ -332,6 +339,21 @@ func absArgs(args []string, flags []string) {
 	}
 }
 
+// relativizePaths converts absolute paths found in the given output string to
+// relative, if they are within the PWD.
```

Comment:
Replace `PWD` here with `working directory`. `PWD` means "print working directory", and since we're not printing it, I don't think we should use that acronym here.

Reply:
TIL! I always thought it was "present working directory". Now the distinction between pwd and cwd makes more sense to me. Thanks